lte/gateway/python/scripts/config_stateless_agw.py

```diff
@@ -165,6 +165,8 @@ def ovs_reset_bridges():
 
 
 def sctpd_pre_start():
+    subprocess.Popen("service procps restart".split())
```

Comment:
should this be a blocking call? currently it will call the service restart and continue with the other process, is this the intended behavior?

Reply:
yes, thats fine. it does not need to block sctpd service start.